code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -905,3 +905,18 @@
 		to_chat(M, "<span class='notice'>[tox_message]</span>")
 	. = 1
 	..()
+
+//Apparently we still haven't shaken the use of asbestos into the year 2303
+/datum/reagent/toxin/asbestos
+	name = "Asbestos"
+	description = "A toxin that lingers in the lungs eventually dealing damage to them when inhaled."
+	silent_toxin = TRUE
+	color = "#fff"
+	metabolization_rate = 0 //It doesn't metabolize. It gets removed when it activates
+	toxpwr = 0
+	taste_description = "Carcinogenic Fireproofing"
+
+/datum/reagent/toxin/asbestos/on_mob_life(mob/living/carbon/M)
+	if(prob(10))
+		M.adjustOrganLoss(ORGAN_SLOT_LUNGS, 15) //Mesothelioma
```

Comment:
This still seems like quite a high amount of damage, and the chat message should only fire once.

Reply:
Ok webedit incoming.